lesson-3/main.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require_relative 'config'
+
+class Main
+  extend MenuPrinter
+  @trains = []
+  @stations = []
+  @routs = []
```

Comment:
[You define instance variables but use in class defined methods.](https://maximomussini.com/posts/ruby-class-variables/)

Reply:
What do you mean?

Reply:
![Screenshot from 2019-07-12 15-27-27](https://user-images.githubusercontent.com/16723263/61128061-abd0be80-a4b9-11e9-8e42-ae54a68563f4.png)